### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by Napassawxn**
This feature is for adding an autocomplete into the search box.

### CODE REVIEW

File: lab-exercise-1/NapassawanFeature.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 public class NapassawanFeature {
     public void feature() {
         // this is a new feature
+
```
Comment (JNapat) [inline lab-exercise-1/NapassawanFeature.java:4]: Function is empty nah ploy

Comment (nekongum) [reply lab-exercise-1/NapassawanFeature.java:4]: Improve pls

Comment (6588041chalisakengkaewpennapa) [reply lab-exercise-1/NapassawanFeature.java:4]: you should fix this code

File: lab-exercise-1/NapassawanFeature.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-public class <Yourname>Feature {
+public class NapassawanFeature {
     public void feature() {
```
Comment (6588041chalisakengkaewpennapa) [inline lab-exercise-1/NapassawanFeature.java:2]: I think this one is good

Comment (JNapat) [reply lab-exercise-1/NapassawanFeature.java:2]: i think soooo

File: lab-exercise-1/NapassawanFeature.java [hunk 3]
```diff
@@ -1,6 +1,7 @@
-public class NapassawanFeature {
+public class <Yourname>Feature {
     public void feature() {
         // this is a new feature
-        
+        int x = 0;
+        x = x+1;
```
Comment (jaxmmmm) [inline lab-exercise-1/NapassawanFeature.java:5]: good ka

**Pull request merged**